### ISSUE TITLE
Learning rate

### ISSUE THREAD

**Issue by kurapan** [sbillburg/CRNN-with-STN#7]
Hi!
I read in a different issue that you are able to achieve 90% accuracy in 24 hours.
I'd like to ask what is the learning rate you used for that. Is it 0.0001 or 0.002? There are two values in the config.py so I was wondering which of the two was used for that.
I'm quite struggling to train the model as I can only reach like 3% accuracy in 48 hours when training on 4 GPUs...

**Comment by sbillburg** [comment]
As long as you don't set the learning rate too small(like 0.00001) or too big(like 1.0), and make sure your network and datasets are correct, the training would be going well. 

I guess your learning rate might set too small?

**Comment by kurapan** [comment]
My bad. I was just being reckless. I normalized the image pixel values to 0-1 range as that is usually recommended in machine learning (rather than using large integer values), but I failed to realize that the array for the batch was created as an integer array. Therefore, I was training pretty much only on images of black background. I don't understand how it was able to achieve even that 3% accuracy xD
Now I've reached 20% accuracy in about an hour (using 4 gpus).
